web/src/components/questions/PackageErrorQuestion.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) [2022-2024] SUSE LLC
```

Comment:
```suggestion
 * Copyright (c) [2022-2025] SUSE LLC
```

Reply:
NP: it's actually a new file, so  

```suggestion
 * Copyright (c) [2025] SUSE LLC
```